File: Up_wpf/Up_wpf/Subject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Up_wpf
{
    public class Subject
    {
        private string name;
        private string aud;
        private string lector;
        private string type;
        public Subject() { }
        public Subject(string name, string aud, string lector, string type)
        {
            this.name = name;
            this.aud = aud;
            this.lector = lector;
            this.type = type;
        }
        public string getAud()
        {
            return aud;
        }

        public string getName()
        {
            return name;
        }

        public string getLector()
        {
            return lector;
        }

        public void setAud(string aud)
        {
            this.aud = aud;
        }

        public void setName(string name)
        {
            this.name = name;
        }

        public void setLector(string lector)
        {
            this.lector = lector;
        }

        public string getType()
        {
            return type;
        }
        public void setType(string type)
        {
            this.type = type;
        }
    }
}

File: Up/app/src/main/java/com/product/dstavrov/up/Helpers/Subject.java
package com.product.dstavrov.up.Helpers;

/**
 * Created by vicru on 20.03.2018.
 */

public class Subject {
    private String name;
    private String aud;
    private String lector;
    public Subject(){}
    public Subject(String name, String aud, String lector){
        this.name = name;
        this.aud = aud;
        this.lector = lector;
    }
    public String getAud() {
        return aud;
    }

    public String getName() {
        return name;
    }

    public String getLector() {
        return lector;
    }

    public void setAud(String aud) {
        this.aud = aud;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setLector(String lector) {
        this.lector = lector;
    }

}
